src/util/value.ts

```diff
@@ -66,7 +66,9 @@ export type EXPERIMENTAL_clipping = {
     };
     distance: number;
   }[];
-  location: LatLngHeight;
+  // for compat
+  location?: LatLngHeight;
+  coordinates?: number[];
```

Comment:
Can you also reflect this in `src/core/mantle/types/values.ts`? I don't want to use this global file in the core module as much as possible because of the confusing import relationship.

Reply:
I fixed it. [fix: clipping type in reearth/core](https://github.com/reearth/reearth-web/pull/377/commits/60437dcdebe4793d7fa9cf3a9e599f9c5caf6cd1)